### PR TITLE
MAINT: Ignore deliberate "adn" typo in README.rst

### PR DESCRIPTION
**Pull request by bl-ue**
Followup to #1535

### CODE REVIEW

**Comment by bl-ue** [comment]
@peternewman what about https://github.com/codespell-project/codespell/blob/skip-adn-readme-followup1535/.github/workflows/codespell.yml?  Should I create a whole new GHA step for just the README? 🤔

**Comment by peternewman** [comment]
> @peternewman what about https://github.com/codespell-project/codespell/blob/skip-adn-readme-followup1535/.github/workflows/codespell.yml? Should I create a whole new GHA step for just the README? thinking

I don't think you need to particularly, and it might confuse people somewhat if they go to use our action.

File: .github/workflows/codespell-private.yml [hunk 1]
```diff
@@ -34,6 +34,7 @@ jobs:
       - run: codespell --help
       - run: make check
       - run: codespell --check-filenames --skip="./.git/*,*.pyc,./codespell_lib/tests/test_basic.py,./codespell_lib/data/*,./example/code.c,./build/lib/codespell_lib/tests/test_basic.py,./build/lib/codespell_lib/data/*,README.rst,*.egg-info/*"
+      - run: codespell README.rst --ignore-words-list="adn"
```
Comment (peternewman) [inline .github/workflows/codespell-private.yml:37]:
```suggestion
      - run: codespell --check-filenames --skip="./.git/*,*.pyc,./codespell_lib/tests/test_basic.py,./codespell_lib/data/*,./example/code.c,./build/lib/codespell_lib/tests/test_basic.py,./build/lib/codespell_lib/data/*,README.rst,*.egg-info/*,.github/workflows/codespell-private.yml"
      - run: codespell --check-filenames --ignore-words-list="adn" README.rst .github/workflows/codespell-private.yml
```

**Review by peternewman — CHANGES_REQUESTED** [review]
You'll need to ignore `.github/workflows/codespell-private.yml` in `.github/workflows/codespell.yml` now. Or better yet explicitly differentiate between `.git/*` and `.github/`

**Comment by bl-ue** [comment]
Oh, funny.  Didn't think of that (didn't check the CI either 😛)

**Pull request closed without merging**